src/material/paginator/paginator.scss

```diff
@@ -69,6 +69,7 @@ $button-icon-size: 28px;
 }
 
 .mat-paginator-icon {
+  display: inline;
```

Comment:
Since it has a `width`, I think that we should use `inline-block`.

Reply:
Done